src/sentry/static/sentry/app/types/hooks.ts

```diff
@@ -230,7 +230,7 @@ type LegacyAnalyticsEvent = (
   /**
    * Arbitrary data to track
    */
-  data: {[key: string]: number | string | boolean}
+  data: {[key: string]: any}
```

Comment:
Not sure why new typescript causes this to fail, but it complains because data be `string[], number[], Aggregation[]`